meta-mender-intel/templates/local.conf.append

```diff
@@ -3,11 +3,10 @@
 
 MACHINE ?= "intel-corei7-64"
 
-# Note that these settings have been verified for booting from a SD card.
-# You might need to change the two latter settings if you are planning to
-# boot from a different storage medium.
+# Note that this setting has been verified for booting from a SD card.
+# You might need to change the setting if you are planning to boot from
+# a different storage medium.
 MENDER_STORAGE_DEVICE = "/dev/mmcblk2"
-MENDER_GRUB_STORAGE_DEVICE = "hd0"
```

Comment:
Commenting out would be nicer, so the variable is still visible.

Reply:
The variable is deprecated, it's causing a fatal error if used ("this is now dynamically determined at runtime."), isn't exposed in the documentation anymore, and it isn't in any other layer.

It seems to me a bad idea as you can't use it in any case. Why would you keep it commented?

Reply:
Ah you're right, sorry misremembered it. Removing it is fine then, of course.